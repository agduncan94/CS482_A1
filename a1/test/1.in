;this is a comment
>header seq1
ACTTTTTGGTCAGATAGAGTGGAGATGAAAGGGTAAA
;this is another comment
ACTTTTTTT
>header seq 2
ATTCT
