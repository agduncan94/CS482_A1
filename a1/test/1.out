AAA 0.0638
AAC 0.0213
AAG 0.0213
AAT 0.0000
ACA 0.0000
ACC 0.0000
ACG 0.0000
ACT 0.0426
AGA 0.0638
AGC 0.0000
AGG 0.0213
AGT 0.0213
ATA 0.0213
ATC 0.0000
ATG 0.0213
ATT 0.0213
CAA 0.0000
CAC 0.0000
CAG 0.0213
CAT 0.0000
CCA 0.0000
CCC 0.0000
CCG 0.0000
CCT 0.0000
CGA 0.0000
CGC 0.0000
CGG 0.0000
CGT 0.0000
CTA 0.0000
CTC 0.0000
CTG 0.0000
CTT 0.0426
GAA 0.0213
GAC 0.0000
GAG 0.0426
GAT 0.0426
GCA 0.0000
GCC 0.0000
GCG 0.0000
GCT 0.0000
GGA 0.0213
GGC 0.0000
GGG 0.0213
GGT 0.0426
GTA 0.0213
GTC 0.0213
GTG 0.0213
GTT 0.0000
TAA 0.0213
TAC 0.0000
TAG 0.0213
TAT 0.0000
TCA 0.0213
TCC 0.0000
TCG 0.0000
TCT 0.0213
TGA 0.0213
TGC 0.0000
TGG 0.0426
TGT 0.0000
TTA 0.0000
TTC 0.0213
TTG 0.0213
TTT 0.1702
